virt/install/preparing-cluster-for-virt.adoc

```diff
@@ -41,6 +41,12 @@ Without an external monitoring system or a qualified human monitoring node healt
 
 * If your cluster uses worker nodes from multiple CPU vendors, live migration failures can occur. For example, a virtual machine with an AMD CPU might attempt to live-migrate to a node with an Intel CPU and likely fail migration. To avoid this, label nodes with a vendor-specific label, such as `Vendor=Intel` or `Vendor=AMD`, and set node affinity on your virtual machines to ensure successful migration. See xref:../../nodes/scheduling/nodes-scheduler-node-affinity.adoc#nodes-scheduler-node-affinity-configuring-required_nodes-scheduler-node-affinity[Configuring a required node affinity rule] for more information.
 
+* All CPUs must have Red Hat Enterprise Linux 8 and support the following requirements:
+
+** Intel 64 or AMD64 CPU extensions
```

Comment:
Suggest rewording the list items for parallel construction

Reply:
Reworded.